pkg/kv/kvserver/batcheval/cmd_gc.go

```diff
@@ -178,6 +182,18 @@ func GC(
 		return result.Result{}, err
 	}
 
+	// Fast path operation to try to remove all user key data from the range.
+	if rk := args.ClearRangeKey; rk != nil {
```

Comment:
We should also check that we're not setting `Keys` or `RangeKeys` when we're processing this.

And at the start of this function, where we're preventing setting the GC threshold when we perform GC operations, we need to guard against `ClearRangeKey` too.